docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-defining-a-stage-proc.adoc

```diff
@@ -1,2 +1,28 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-defining-a-stage-proc']
+= Defining a stage
+
+A stage can be modeled in BPMN2 using the case designer. Stages are a way of grouping related tasks in a way that clearly define activities that must complete before the next stage of the case commences. For example, the IT Orders case definition described in <<case-management-designing-a-case-proc>> can also be defined using stages in the following way:
+
+.IT Orders project stages example
+image::itorders-stages.png[IT Orders - stages]
+
+.Prerequisites
+
+* You have followed the procedure in <<case-management-creating-a-case-proc>> to create a new case in {CENTRAL}.
+* You have started a case definition and defined any other required case nodes described in <<case-management-designing-a-case-proc>>. Stages must be triggered by another node within a case definition, therefore at least one other node must be configured in the designer before the stage can be started.  
+
+.Procedure
+. From the case designer, click image:3897.png[] on the left of the palette to open the *Object Library*. Open the  *Subprocesses* object list, drag and drop an *Ad-Hoc* node onto the design canvas and provide a name for the stage node.
+. Define how the stage is activated: 
++ 
+* If the stage is being activated by an incoming node, connect the stage with a sequence flow line from the incoming node.
+* If the stage is instead being triggered by a signal event, configure the `SignalRef` on the signal node with the name of the stage that you configured in the first step. 
+* Alternatively, configure the `AdHocActivationCondition` property to start the stage when the condition has been met.
+
+. Re-size the node as required to allow room to add the task nodes for the stage. 
+. Add the relevant tasks to the stage and configure them as required.
+. By default, stages are configured as `autocomplete`, which means that the stage will automatically complete and trigger the next activity in the case definition once all instances in the stage are no longer active. 
```

Comment:
Bits "By default" and "To change" should both not be numbered, imo. The latter isn't, and the former shouldn't be either.

Reply:
I've changed this to make it clearer. The "By default" is actually about configuring the stage, where the previous step was about configuring the tasks within the stage, so two different things. I think I meant to come back and put an actual step before the explanation and forgot.